### PR TITLE
Billing History: Add comments to translation variables

### PR DESCRIPTION
**Pull request by tyxla**
See p7H4VZ-2fo-p2 for motivation of this PR.

#### Changes proposed in this Pull Request

* Add comments to billing period translation variables to provide context for translators.

#### Testing instructions

* Verify code looks sane.

### CODE REVIEW

File: client/me/purchases/manage-purchase/plan-details/billing-period.jsx [hunk 1]
```diff
@@ -51,12 +51,14 @@ class PlanBillingPeriod extends Component {
 		if ( isRenewing( purchase ) && purchase.renewMoment ) {
 			return translate( 'Billed yearly, renews on %s', {
 				args: purchase.renewMoment.format( 'LL' ),
+				comment: '%s the renewal date in format M DD, Y, for example: June 10, 2019',
```
Comment (jsnajdr) [inline client/me/purchases/manage-purchase/plan-details/billing-period.jsx]: `%s` could be the subject of a well-formed sentence:
```
%s is the renewal date in format M DD, Y, e.g., June 10, 2019
```

**Review by jsnajdr — APPROVED** [review]
Looks good, I just posted a little grammar nit 😉

File: client/me/purchases/manage-purchase/plan-details/billing-period.jsx (hunk 1 above)
Comment (tyxla) [reply client/me/purchases/manage-purchase/plan-details/billing-period.jsx]: 🤦‍♂ 

That's what I was trying to do actually. Thanks, fixing now 😄

Comment (tyxla) [reply client/me/purchases/manage-purchase/plan-details/billing-period.jsx]: Fixed in e60c036874528d1acc1b67f2ed30ec6a5ebb5318

**Comment by lsl** [comment]
I see your merged but FYI:

`comment` changes like this are perfectly fine and do not trigger a re-translation. `context` on the other hand is used to create a unique key with the string text, so changing or adding a context would trigger re-translation.

**Comment by tyxla** [comment]
Those were exactly my intentions, thanks for confirming @lsl 🙌

**Pull request merged**